src/test/java/com/hydratereminder/chat/ChatMessageSenderTest.java

```diff
@@ -28,47 +28,46 @@ class ChatMessageSenderTest {
     @InjectMocks
     private transient ChatMessageSender chatMessageSender;
 
+    private static final String  message = "Hello";
```

Comment:
Can you remove the extra white-space between `String` and `message`?

Let's also make the variable name more descriptive like `chatMessageToSend`.